pkg/konnector/healthz/server.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2023 The Kube Bind Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package healthz
+
+import (
+	"context"
+	"net"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"k8s.io/klog/v2"
+)
+
+const listenAddr = ":8080"
+
+type Server struct {
+	Router  *mux.Router
+	Checker checker
+}
+
+func NewServer() (*Server, error) {
+	server := &Server{
+		Router:  mux.NewRouter(),
+		Checker: checker{},
+	}
+
+	return server, nil
+}
+
+func (s *Server) Start(ctx context.Context) {
+	s.Router.Handle("/healthz", s.Checker)
+	server := &http.Server{
+		Handler: s.Router,
+	}
+	go func() {
+		listener, err := net.Listen("tcp", listenAddr)
+		if err != nil {
+			panic(err)
+		}
+		server.Serve(listener) //nolint: errcheck
+	}()
+
+	log := klog.FromContext(ctx)
+	log.Info("healthz endpoint listening", "address", server.Addr)
```

Comment:
```suggestion
	log.Info("healthz endpoint listening", "address", listenAddr)
```

`server.Addr` is empty unless we set it, so currently the logline looks like this:
```
server.go:58] "healthz endpoint listening" address=""
```